build.rs

```diff
@@ -297,7 +297,7 @@ mod c {
             }
         }
 
-        if target_arch == "arm" && target_os != "ios" {
+        if target_arch == "arm" && target_os != "ios" && target_env != "msvc" {
```

Comment:
There's some C files below in addition to assembly files, should they be included even on MSVC?

Reply:
My understanding is that LLVM uses different builtins when targeting the msvc subsystem, for example `__rt_udiv` instead of `__aeabi_uidivmod`. I don't think the C files are required, and I haven't hit any errors yet due to their absence. I checked a few of the C files and they contain helper code for the assembly code.

Reply:
Ok!